tests/test_hash.py

```diff
@@ -259,7 +250,7 @@ def test_update_file_obj(self):
 
   def _do_update_file_obj(self, library, algorithm):
     data = 'abcdefgh' * 4096
-    file_obj = six.StringIO()
+    file_obj = io.StringIO()
```

Comment:
I did not even know this existed :eyes: ... seems nice

Reply:
same here